server/views/temporary-accommodation/static/notAuthorised.njk

```diff
@@ -11,10 +11,19 @@
         <ul class="govuk-list govuk-list--bullet">
           <li>the CRN is not in your caseload</li>
           <li>referrals through the Transitional Accommodation (CAS3) service are not currently available in your region</li>
+          <li>the home area on your NDelius account is incorrect</li>
         </ul>
       </p>
+      <h2 class="govuk-heading-m">Incorrect home area on NDelius account</h2>
       <p class="govuk-body">
-        If this is incorrect contact <a href="mailto:cas3support@digital.justice.gov.uk">cas3support@digital.justice.gov.uk</a>
+        If your home area is incorrect on NDelius the service will not recognise your account as being in the correct region.
```

Comment:
@selina-dxw I wondered if it would be helpful to point out that their current region is set in NDelius as. It's what we're displaying in the top right of the page. We might be able present repeat that in this paragraph to provide more clarity as I'm not sure users have access to see what their ndelius account is set to.

"Your home area is currently set to 'West Midlands' within NDelius. If this is incorrect this service may not recognise your account as being in the correct region."

Reply:
That said, we're planning to rollout to all regions soon so this page should be shown less and less.

Reply:
Yeah I like that, I think it would be good to add even if we're rolling out to more regions soon. 

Could we also hyperlink 'Technology Portal' to: https://mojprod.service-now.com/moj_sp?id=sc_cat_item&table=sc_cat_item&sys_id=21952b8edb73e7441b4ffc45ae96197b